openmetadata-ui/src/main/resources/ui/src/locale/languages/en-us.json

```
@@ -689,6 +689,7 @@
     "open-lowercase": "open",
     "open-metadata": "OpenMetadata",
     "open-metadata-logo": "OpenMetadata Logo",
+    "open-metadata-updated": "OpenMetadata Updated !",
```

```suggestion
    "open-metadata-updated": "OpenMetadata Updated!",
```
I think space is not necessary here, @ShilpaVernekar any thoughts?